pints/plot/_surface.py

```diff
@@ -0,0 +1,324 @@
+#
+# Plots a function defined on a two-dimensional space.
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+# This module was was adapted from Myokit (see http://myokit.org) by the
+# original author.
+#
+# The code to plot voronoi regions was based on an example shown here:
+# http://stackoverflow.com/a/20678647/423420
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+import numpy as np
+
+import pints
+
+
+def surface(points, values, boundaries=None, markers='+', figsize=None):
+    """
+    Takes irregularly spaced points and function evaluations in a
+    two-dimensional parameter space and creates a coloured surface plot using a
+    voronoi diagram.
+
+    Returns a ``matplotlib`` figure object and axes handle.
+
+    Parameters
+    ----------
+    points
+        A list of (two-dimensional) points in parameter space.
+    values
+        The values corresponding to these points.
+    boundaries
+        An optional :class:`pints.RectangularBoundaries` object to restrict the
+        area shown.
+    markers
+        The markers to use to plot the sampled points. Set to ``None`` to
```

Comment:
Maybe say 'matplotlib' markers? I wouldn't know what possible markers would be available.